doc/_static/custom_styles.css

```
@@ -160,3 +165,66 @@ div#driver-popover-item .tour-progress-indicator {
 a[href="#join-our-community"] + ul li.toc-h3 {
   display: none;
 }
+
+.dropdown-videos {
+  width: 100%;
+  max-width: 640px;
+  margin-left: auto;
+  margin-right: auto;
+  border: 0;
+  border-radius: unset;
+  margin-top:-1px;
+}
+
+.dropdown-videos .summary-content {
+  background-color: #EDEDED;
```

```suggestion
  background-color: #ededed;
```

Lowercase is the standard we use elsewhere